gpii/node_modules/gpii-oauth2/gpii-oauth2-authz-server/webTests/core/html/GPIIOauth2-test.html

```diff
@@ -18,6 +18,15 @@
     <script type="text/javascript" src="../../../../../../../node_modules/infusion/tests/test-core/jqUnit/js/jqUnit.js"></script>
     <script type="text/javascript" src="../../../../../../../node_modules/infusion/tests/test-core/jqUnit/js/jqUnit-browser.js"></script>
 
+    <!-- These are required not for the tests themselves, but for the coverage client -->
+    <script type="text/javascript" src="../../../../../../../node_modules/infusion/src/framework/core/js/DataBinding.js"></script>
```

Comment:
Yes.  They were not needed in other files that had already included them for their own tests.